reana/cli.py

```diff
@@ -327,15 +327,26 @@ def find_standard_component_name(short_component_name):
 
     :raise: exception in case more than one is found
     """
-    output = []
-    for component in REPO_LIST_ALL:
-        component_short_name = shorten_component_name(component)
-        if component_short_name == short_component_name:
-            output.append(component)
-    if len(output) == 1:
-        return output[0]
-    raise Exception('Component name {0} cannot be uniquely mapped.'.format(
-        'short_component_name'))
+    def _is_standard_name(component_name):
+        """Detect whether the provided component name is already standard."""
+        prefixes = component_name.split('-')[:-1]
+        return all([len(n) > 1 for n in prefixes])
+
+    if _is_standard_name(short_component_name):
+        standard_component_name = short_component_name
+    else:
+        output = []
+        for component in REPO_LIST_ALL:
+            component_short_name = shorten_component_name(component)
+            if component_short_name == short_component_name:
```

Comment:
`component_short_name` vs `short_component_name`? What's the difference? I think the naming can be improved.

Also, correct me if I'm wrong, the argument that this function is receiving is called `short_component_name` but I see that it's first checked if this is already a standard name (`is_standard_name`), in case this is true then the name of the variable is confusing. I think the description of the function can be amended too as now it can receive a non-short component name.